cli/server.ts

```diff
@@ -99,41 +99,44 @@ function readAssetsAsync(logicalDirname: string): Promise<any> {
         }))
 }
 
-function readPkgAsync(logicalDirname: string, fileContents = false): Promise<FsPkg> {
+async function readPkgAsync(logicalDirname: string, fileContents = false): Promise<FsPkg> {
     let dirname = path.join(userProjectsDir, logicalDirname)
-    let r: FsPkg = undefined;
-    return readFileAsync(path.join(dirname, pxt.CONFIG_NAME))
-        .then(buf => {
-            let cfg: pxt.PackageConfig = JSON.parse(buf.toString("utf8"))
-            let files = [pxt.CONFIG_NAME].concat(cfg.files || []).concat(cfg.testFiles || [])
-            return Promise.map(files, fn =>
-                statOptAsync(path.join(dirname, fn))
-                    .then<FsFile>(st => {
-                        let r: FsFile = {
-                            name: fn,
-                            mtime: st ? st.mtime.getTime() : null
-                        }
-                        if (st == null || !fileContents)
-                            return r
-                        else
-                            return readFileAsync(path.join(dirname, fn))
-                                .then(buf => {
-                                    r.content = buf.toString("utf8")
-                                    return r
-                                })
-                    }))
-                .then(files => {
-                    r = {
-                        path: logicalDirname,
-                        config: cfg,
-                        files: files
-                    };
-                    return existsAsync(path.join(dirname, "icon.jpeg"));
-                }).then(icon => {
-                    r.icon = icon ? "/icon/" + logicalDirname : undefined;
-                    return r;
-                })
-        })
+    let buf = await readFileAsync(path.join(dirname, pxt.CONFIG_NAME))
+    let cfg: pxt.PackageConfig = JSON.parse(buf.toString("utf8"))
+    let r: FsPkg = {
+        path: logicalDirname,
+        config: cfg,
+        files: []
+    };
+    
```

Comment:
Linting error on this line: `ERROR: /home/travis/build/Microsoft/pxt/cli/server.ts[111, 1]: trailing whitespace`

Reply:
Right, fixed these, thanks!